lib/btn/btn.scss

```diff
@@ -0,0 +1,36 @@
+@import "palette.scss";
+@import "mixins/btn.scss";
+
+//button variants
+.btn {
+  @include btn();
+}
+
+.btn--light {
+  @include btn($color-400-grey);
+}
+
+.btn--active {
+  @include btn($btn-active-bg-color);
+}
+
+.btn--danger {
+  @include btn($danger-color);
+}
+
+.btn--mortal-danger {
+  @include btn($mortal-danger-color);
+}
+
+//button sizes
+.btn--small {
+  @include btn-size("small");
+}
+.btn--large {
+  @include btn-size("large");
+}
+
+.btn--very-large {
```

Comment:
Couldn't we rename them in Composer ahead of time? Should be a simple Find & Replace job...?

We shouldn't have to let legacy naming hold back improvements.